docs/jwst/undersampling_correction/description.rst

```diff
@@ -6,21 +6,22 @@ Description
 
 Overview
 --------
-This step corrects for an artifact seen in undersampled NIRISS images taken during commissioning.
-The issue seen was that peak pixels of stars which reach close to saturation in dithers for which
-the star is centered on the center of a pixel suffer from relatively strong charge migration such
-that the group-to-group differences decrease significantly up the ramp once the signal level is
-greater than roughly 30,000 ADU.  As a result, the last several groups of these ramps get flagged
-by the ``jump`` step. The smaller number of groups used for these pixels in the ramp_fitting step
-results in them having larger read noise variances, which in turn leads to lower weights used
-during resampling. This ultimately leads to a lower than normal flux for the star in resampled
-images.
-
-Once a group in a ramp has been flagged as affected by charge migration, all subsequent groups
-in the ramp are also flagged. By flagging these groups, they will not get used in the
-computation of slopes in the ``ramp_fitting`` step, and as described in the algorithm section below,
-they will be used in the calculation of the variance of the slope due to readnoise.
-
+This step corrects for an artifact seen in undersampled NIRISS images that may depress flux 
+in resampled images. The artifact is seen in dithered images where the star is centered in 
+a pixel. When the peak pixels of such stars approach the saturation level, they suffer from 
+significant :ref:`charge migration <charge_migration>`:
+the spilling of charge from a saturated pixel into its neighboring pixels. This charge migration 
+causes group-to-group differences to decrease significantly once the signal level is greater than 
+~30,000 ADU. As a result, the last several groups of these ramps get flagged by the ``jump`` step. 
+The smaller number of groups used for these pixels in the ``ramp_fitting`` step results in them having 
+larger read noise variances, which in turn leads to lower weights used during resampling. This 
+ultimately leads to a lower than normal flux for the star in resampled images.
+
+Once a group in a ramp has been flagged as affected by charge migration, all subsequent 
+groups in the ramp are also flagged. By flagging these groups, they are not used in the 
+computation of slopes in the :ref:`ramp_fitting <ramp_fitting>` step. However, as described 
```

Comment:
This reference is causing the build to fail. Where is the section located?

Reply:
The name of the section was incorrect, I have fixed it